kwave/kgrid.py

```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 
-from kwave.data import Array
+from kwave.data import Vector, FlexibleVector
```

Comment:
I don't understand the difference in use-case between Vector and FlexibleVector at first glance.

Reply:
We had a offline discussion after the comment but putting the idea here for posterity. `Vector` is designed to contain elements of the same type while `FlexibleVector` can contain mixture of element types. Ideally, we don't want `FlexibleVector` functionality. It is deprecated and will be removed once we refactor the `kWaveGrid` object. It is the only place where `FlexibleVector` is used.